maintainers/maintainer-list.nix

```diff
@@ -1252,6 +1252,16 @@
     githubId = 3043718;
     name = "Brett Lyons";
   };
+  brodes = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add brodes``.